pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -259,9 +259,12 @@ def test_order_without_freq(self, index_dates, expected_dates, tz_naive_fixture)
         tm.assert_numpy_array_equal(indexer, exp, check_dtype=False)
         assert ordered.freq is None
 
-    def test_drop_duplicates_metadata(self):
+    @pytest.mark.parametrize(
```

Comment:
would be ok with making this a fixture, call it freq_sample, and put in pandas/tests/indexes/conftest.py